### PR TITLE
Re-adds ashwalker ash resistance

### PR DESCRIPTION
**Pull request by ghost**
Honestly i'm not sure why the last pr was merged it was a joke

##### Changelog

:cl:
rscadd: ashwalkers have ash resistance, goddamnit cruix it was a joek PR
/:cl:

### CODE REVIEW

**Comment by oisin100** [comment]
I dont get it.

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -80,6 +81,7 @@
 		var/mob/living/carbon/human/H = new_spawn
 		H.languages_spoken |= HUMAN
 		H.languages_understood |= HUMAN
+		H.weather_immunities |= "ash"
```
Comment (oisin100) [inline code/game/objects/structures/ghost_role_spawners.dm]: This code isnt needed, It gets ran already by its parent proc

**Review by oisin100 — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/ghost_role_spawners.dm (hunk 1 above)
Comment (ghost) [reply code/game/objects/structures/ghost_role_spawners.dm]: they originally had it as well, oisin, as well as the fact that in testing the chieftain dies in ash without this code

Comment (Super3222) [reply code/game/objects/structures/ghost_role_spawners.dm]: Ash Immunity is already added in the parent proc. Oisin is right, and no - this wasn't in the original code either.

Comment (oisin100) [reply code/game/objects/structures/ghost_role_spawners.dm]: Original code: http://prntscr.com/dry2t9

**Comment by ShadowDeath6** [comment]
fix your shit

**Review by oisin100 — APPROVED** [review]

**Pull request merged**